sormas-app/app/src/main/java/de/symeda/sormas/app/report/aggregate/AggregateReportsFragment.java

```diff
@@ -374,7 +372,9 @@ public void doInBackground(TaskResultHolder resultHolder) throws DaoException {
 						if (report.getLocalChangeDate() == null
 							&& (report.getNewCases() == null || report.getNewCases() == 0)
 							&& (report.getLabConfirmations() == null || report.getLabConfirmations() == 0)
-							&& (report.getDeaths() == null || report.getDeaths() == 0)) {
+							&& (report.getDeaths() == null || report.getDeaths() == 0)
+							&& report.getRegion() == null
```

Comment:
Does it make sense to include region and district here? I think they should be checked before (i.e. submitting the numbers should with no region and district filled in should result in an error); right here, they rather water down the logic

Reply:
it will be removed